access.go

```diff
@@ -47,6 +47,12 @@ type AccessRequest struct {
 	// Token expiration in seconds. Change if different from default
 	Expiration int32
 
+	// Refresh Token expiration in seconds. Change if different from default
+	RefreshExpiration int32
+
+	// Extend Refresh Token expiration
+	ExtendRefreshExpiration bool
```

Comment:
I think we need it. if we don't decode RefreshToken jwt we can't got the extended value since for now we produce the expiration in Generator.go.

@rexydwybrundy